app/views/providers/cookies/show.html.erb

```diff
@@ -43,7 +43,7 @@
     <p class="govuk-body"><%= t(".these_essential_cookies") %></p>
     <p class="govuk-body"><%= t(".they_always_need") %></p>
 
-    <%= link_to_accessible(t(".find_out_more_link"), "https://www.gov.uk/help/cookie-details", class: "govuk-link govuk-body") %>
+    <%= govuk_link_to(t(".find_out_more_link"), "https://www.gov.uk/help/cookie-details", class: "govuk-body") %>
```

Comment:
Not your fault at all, because this was existing, but I think this is better as:
```
<p class='govuk-body'>
    <%= govuk_link_to(t(".find_out_more_link"), "https://www.gov.uk/help/cookie-details") %>
</p>
```
And this pattern should be used for any of the links that have a "govuk-body" class. Though this is not critical and is just a suggestion as you haven't made these changes. Not a blocking issue